classes/services/queryBuilders/IssueListQueryBuilder.inc.php

```diff
@@ -154,8 +155,15 @@ public function get() {
 		$q = Capsule::table('issues as i')
 					->where('i.journal_id','=', $this->contextId)
 					->leftJoin('issue_settings as is', 'i.journal_id', '=', 'is.issue_id')
-					->orderBy($this->orderColumn, $this->orderDirection)
-					->groupBy('i.issue_id');
+					->orderBy($this->orderColumn, $this->orderDirection);
+
+		$isSqlServer = Config::getVar('database', 'ms_sql');
+		if ($isSqlServer) {
+		    $q->groupBy('i.issue_id', 'i.journal_id', 'i.volume', 'i.number', 'i.year', 'i.published', 'i.actual', 'i.date_published', 'i.date_notified', 'i.last_modified', 'i.access_status', 'i.open_access_date', 'i.show_volume', 'i.show_number', 'i.show_year', 'i.show_title', 'i.style_file_name', 'i.original_style_file_name');
```

Comment:
Older versions of PostgreSQL had a similar issue where they weren't smart enough to realize that `GROUP BY` on a primary key legitimized selection of that table's columns. Do you know whether there's some DDL that would inform SQL server of that possibility?

Reply:
I don't think it is possible with SQL server, I agree it is kinda cumbersome. We will change this if we find something useful throughout the integration process.

Reply:
@NateWr, mind taking a look at the above comment?

Reply:
I'm sure that we'll be joining `issue_settings` once we start building out that API endpoint with actual use-cases, but it's fine to remove for now.

However, the following line:

```
$q->orderBy($this->orderColumn, $this->orderDirection);
```

Should be moved above the hook:

```
\HookRegistry::call('Issue::getIssues::queryObject', array(&$q, $this));
```

So that the ordering parameters can be modified by a plugin.